### PR TITLE
172 migrate static controller spec to request spec

### LINKED ISSUE #172: Migrate Static Controller Spec to Request Spec
The current attitudes are to move away from Controller specs in favor of Request specs. We have some Request specs started for our API.

Migrate `/spec/controllers/static_controller_spec.rb` to request spec

## Reference
 - https://relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec
 - https://everydayrails.com/2016/08/29/replace-rspec-controller-tests.html
 - https://github.com/rubyforgood/diaper/issues/739

### PR DESCRIPTION
**Pull request by jeduardo824**
Resolves #172  

### Description

I just moved the file `static_controller_spec` to `static_request_spec`. I made the necessary changes and I add a prefix (static_page_path) to `static/pages` to `routes.rb` file

### Type of change

* Improvement (?)

### How Has This Been Tested?

All the tests were green.

### CODE REVIEW

File: spec/requests/static_request_spec.rb [hunk 1]
```diff
@@ -1,34 +1,37 @@
...
       it "returns http success" do
-        get :index
+        get root_path
         expect(response).to have_http_status(200)
       end
     end
   end
   context "when url is correct and template is available" do
     describe "GET #page" do
+      let(:name) { "index" }
```
Comment (benreyn) [inline spec/requests/static_request_spec.rb]: Im confused as to why this variable is defined in a `let` block. It doesnt appear to be re-used anywhere. Would it be better served to just define it as a plain old ruby variable in the `it` block below?

**Comment by benreyn** [comment]
Thanks so much for this contribution @jeduardo824! Let me know what you think about my comment and Ill get this merged in! 😄

File: spec/requests/static_request_spec.rb (hunk 1 above)
Comment (jeduardo824) [reply spec/requests/static_request_spec.rb]: This is a bit subjective. I think it makes the spec easier to read. I like the organization of defining all my dependent objects with let and keeping my it block nice and short.

Comment (benreyn) [reply spec/requests/static_request_spec.rb]: Agreed on the subjective nature of this comment. This isnt really a blocker to getting this merged 👍

**Pull request merged**